src/pages/Profile.js

```diff
@@ -355,22 +356,131 @@ const PassportLinker = ({ queryParams: { state, code } = {}, provider, prettyNam
+  const { workspaces } = useWorkspaces()
+  const [prefsData] = _.over(_.pickBy)((_v, k) => _.startsWith('notifications/', k), authStore.get().profile)
+
+
+  return h(PageBox, { role: 'main', style: { flexGrow: 1 }, variant: PageBoxVariants.LIGHT }, [
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, margin: 0, textTransform: 'uppercase' } }, [
+        'Account Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } },
+          'You may receive email notifications regarding certain events in Terra. You may globally opt in or out of receiving these emails.'
+        )
+      ])
+    ]),
+    h(NotificationCardHeaders),
+    div({ role: 'list', 'aria-label': 'notification settings for your account', style: { flexGrow: 1, width: '100%' } }, [
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Group Access Requested',
+        notificationKeys: ['notifications/GroupAccessRequestNotification']
+      }),
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Workspace Access Added',
+        notificationKeys: ['notifications/WorkspaceAddedNotification']
+      }),
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Workspace Access Removed',
+        notificationKeys: ['notifications/WorkspaceRemovedNotification']
+      })
+    ]),
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, marginTop: '2rem', textTransform: 'uppercase' } }, [
+        'Submission Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } },
+          'You may receive email notifications regarding terminal submissions in Terra (i.e. if a submission has succeeded, failed, or been aborted). You may opt in or out of receiving these emails for individual workspaces.'
+        )
+      ])
+    ]),
+    h(NotificationCardHeaders),
+    div({ role: 'list', 'aria-label': 'notification settings for workspaces', style: { flexGrow: 1, width: '100%' } },
+      _.map(workspace => {
+        const label = `${workspace.workspace.namespace}/${workspace.workspace.name}`
+        return h(NotificationCard, {
+          setSaving,
+          prefsData,
+          label,
+          notificationKeys: [`notifications/SuccessfulSubmissionNotification/${label}`, `notifications/FailedSubmissionNotification/${label}`, `notifications/AbortedSubmissionNotification/${label}`]
+        })
+      })(workspaces)
+    )
+  ])
+}
+
+const PersonalInfoTab = ({ setSaving }) => {
   const [profileInfo, setProfileInfo] = useState(() => _.mapValues(v => v === 'N/A' ? '' : v, authStore.get().profile))
   const [proxyGroup, setProxyGroup] = useState()
-  const [saving, setSaving] = useState()
+  const { researchArea } = profileInfo
+  const [areasOfResearch, setAreasOfResearch] = useState(_.isEmpty(researchArea) ? [] : _.split(',', researchArea))
```

Comment:
Since `researchArea` is part of `profileInfo` and `profileInfo` is stored in state, is there a need for a separate state variable here? With two state variables, they may get out of sync.

Reply:
Good call- removed the extra state in b6c918f788607a70867e28791012ced4e4d48857